components/ui-api-layer/docs/proposals/exceed-resource-quota.md

```diff
@@ -0,0 +1,126 @@
+# Checking when resource quota has been exceeded proposal
+
+## Overview
+
+We have a few ideas how to check if resource quota exceeded its limit.
+
+## Checking for conditions
+
+We can use deployment's or replica set's conditions to judge if given resource quota has been exceeded.
+
+These are the conditions from:
```

Comment:
`The conditions for ReplicaSet are as follows:`

Reply:
Done